### DISCUSSION TITLE
Using png file as background image?

### DISCUSSION THREAD

**Discussion by WheresWes** [HASwitchPlate/openHASP#373] [Q&A] [answered]
Hi Team,

I have just started playing around with the setup of the WT-SC01 having managed to get everything connected and talking to HA.

I am trying to use an image as a background and have other items appear over the top of it.  I have had the text working (time temperature etc) but when I add the image the text no longer appears - presumably they now sit beneath the image.

Here is my basic code 

{"page":0,"id":2,"obj":"img","src":"/littlefs/back.png"}
{"page":0,"id":1,"obj":"label","x":240,"y":5,"h":30,"w":70,"text":"00.0","align":2,"bg_color":"#008080"}
{"page":0,"id":3,"obj":"label","x":3,"y":5,"h":30,"w":62,"text":"00:00","align":0,"bg_color":"#008080"}
p0b2.to_back

Is what I am trying to do possible?
Thanks W

**Comment by fvanroie** [comment, marked as answer]
Page 0 is a special overlay that is drawn on top of all other pages. Adding an image to page 0 will indeed cover anything on pages 1-12.
Try using page 1 instead.

`to_back` is not needed when the object is the first one on the page or you can use `{"page":1,"id":2,"obj":"img","src":"L:/back.png";"to_back":1}` if needed.

**Reply by WheresWes** [reply]
Thanks.. I got it working.. It turns out that it was doing what I wanted the whole time but that the text happened to be the exact same color as the background!  
Thanks for your help. big fan of the project!
